parse_data.py

```diff
@@ -8,14 +8,43 @@ def read_data(output_f: str):
     session = vk.Session(access_token=settings.VK_AUTH_TOKEN)
     vk_api = vk.API(session)
     nakur = True
+    group_domain = "kalikfan"
     offset = 0
     records = []
+    group = vk_api.groups.getById(v="5.21", group_ids=group_domain)[0]
+    group_id = "-" + str(group["id"])  # Group id must starts with "-"
+
     while nakur:
-        wall = vk_api.wall.get(v="5.21", domain="kalikfan", offset=offset, count=100)
+        wall = vk_api.wall.get(v="5.21", domain=group_domain, offset=offset, count=100)
+
         if len(wall["items"]) != 100:
             nakur = False
         for item in wall["items"]:
+            text = item["text"]
+            # Skip ads
+            if "#калик_рекламик" in text:
+                continue
             records.append(item["text"])
+
+            post_id = str(item["id"])
+            comments_nakur = True
+            comments_offset = 0
+            post_comments = []
+            while comments_nakur:
+                get_comments = vk_api.wall.getComments(v="5.21", owner_id=group_id, post_id=post_id, need_likes=1, count=100, offset=comments_offset)
```

Comment:
same here with the size of list of comments, if you change it to 50 in one place, need to remember to do it in others as well